### PR TITLE
[libcommhistory] Add missing lcCommHistory category in declarative.

### PR DESCRIPTION
**Pull request by dcaliste**
It should fix the issue I created in the declarative part, replacing the old QDEBUG() call without providing an implementation for the lcCommHistory symbol.

### CODE REVIEW

**Comment by dcaliste** [comment]
@pvuorela, I didn't export the symbol, but created a new one, as you suggested. I kept the category name though to ease debugging, so commhistory debug messages come for lib and declarative. If you prefer to define something like com.nokia.commhistory.declarative, I can amend the commit.

File: declarative/src/debug.h [hunk 1]
```diff
@@ -23,14 +23,8 @@
 #ifndef DEBUG_H
 #define DEBUG_H
```
Comment (pvuorela) [inline declarative/src/debug.h:24]: There's nothing using this file now? I guess this was used earlier and you could enable debug only for qml side.

Comment (dcaliste) [reply declarative/src/debug.h:24]: Ah, right, still the includepath = ../src that gets the one from the lib sources. I think it's better not to include the one from the lib sources, so I'm keeping the file, and I've checked that the #include in the declarative sources are including "debug.h" and not "debug_p.h" which the one from the lib.

File: declarative/src/debug.cpp [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#include "debug_p.h"
```
Comment (pvuorela) [inline declarative/src/debug.cpp]: This should be then also debug.h?

Comment (dcaliste) [reply declarative/src/debug.cpp]: Correct : /

**Comment by dcaliste** [comment]
Ah, one other thing also : I've left qWarning as they were. I should change them for qCWarning also. I'll add a commit on top for them.

**Comment by dcaliste** [comment]
Ok, commit added for warnings also.

File: declarative/src/callproxymodel.cpp [hunk 3]
```diff
@@ -1,6 +1,7 @@
 #include "callproxymodel.h"
 #include "event.h"
 #include "eventmodel.h"
+#include "debug_p.h"
```
Comment (pvuorela) [inline declarative/src/callproxymodel.cpp]: debug.h on this side? Same on draftevent and mmshelper.

Comment (dcaliste) [reply declarative/src/callproxymodel.cpp]: Oh sorry. It seems I'm not properly awaken this morning.

**Review by pvuorela — APPROVED** [review]
Looking good now.

**Pull request merged**